pkgs/development/python-modules/should-dsl/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "should_dsl";
```

Comment:
```suggestion
  pname = "should-dsl";
```

Reply:
Thanks, fixed now! I just used the "official" PyPI name here as I thought it should be used here, although I used the normalised name elsewhere.